backend/DB.ts

```diff
@@ -1,24 +1,28 @@
 import Umzug from "umzug";
-import s from "umzug/lib/storages/SequelizeStorage.js";
+import s from "sequelize";
+import ss from "umzug/lib/storages/SequelizeStorage.js";
 import { router } from "./Router.js";
 // @ts-ignore
 import db from "../models/index.cjs";
 
-const { default: SequelizeStorage } = s;
+const { default: SequelizeStorage } = ss;
+const { Sequelize } = s;
 
 export const sequelize = db.sequelize;
 
 export const dbReady = new Promise<void>((resolve, reject) => {
   const intervalId = setInterval(() => {
     console.log("Attempting to connect to db");
     sequelize.authenticate().then(
-      () => {
+      async () => {
         console.log("Database connection established");
         clearInterval(intervalId);
 
         const umzug = new Umzug({
           migrations: {
-            glob: "migrations/*.js",
+            path: "migrations",
+            pattern: /\.cjs$/,
+            params: [sequelize.getQueryInterface(), Sequelize],
```

Comment:
I suspect when we were pairing on your computer, you had already created the nouns table and we weren't relying on this migration - because when I tried to set it up on mine, it didn't work. I had to add these configuration options for it to find the migration file successfully and to pass the expected parameters to the sequelize migration

Reply:
yeah i renamed the files to cjs after the migration ran.